homeassistant/components/jvc_projector/strings.json

```
@@ -37,6 +37,18 @@
       "jvc_power": {
         "name": "[%key:component::sensor::entity_component::power::name%]"
       }
+    },
+    "sensor": {
+      "jvc_power_status": {
+        "name": "Power status",
+        "state": {
+          "standby": "StandBy",
```

I would use as many existing translations as possible (defined in `homeassistant/strings.json`).

```suggestion
          "standby": "[%key:common::state::standby%]"
```